src/main/java/rps/RandomPlayer.java

```diff
@@ -0,0 +1,26 @@
+package rps;
+
+import java.util.Random;
+
+import static rps.Gesture.getGestures;
+import static rps.Gesture.withId;
+
+public class RandomPlayer implements Player {
+    private static final int UPPER_BOUND = getGestures().size();
```

Comment:
With the method I proposed earlier (`numberOfGesturesInGame()`) you would not need to get the gestures and then ask for the size.

Also `UPPER_BOUND` could be renamed to `NUMBER_OF_GESTURES_IN_GAME` (longer, but maybe clearer)